app.js

```diff
@@ -48,22 +49,32 @@ function addClass(insertedDiv, letter, i) {
 }
 
 function gameSetup() {
+  addLetterListeners();
+  insertNextImg();
+  let rndPickedProverb = proverbs[getRandomInt(proverbs.length)];
+  sentence.innerHTML = encryptSentence(rndPickedProverb);
+}
+
+function addLetterListeners() {
   const letters = document.querySelectorAll('.letter-wrapper');
-  [...letters].forEach(item => item.addEventListener('click', function(item) {
-    let pickedLetter = item.target.innerHTML;
-    if (pickedLetter.charAt(0) === '<') {
-      letterCheck(pickedLetter.charAt(3));
-    } else {
-      letterCheck(pickedLetter);
-    }
+  [...letters].forEach(item => item.addEventListener('click', (event) => {
+    letterCheck(item.id);
   }));
-  imageWrapper.innerHTML = '';
+}
+
+function insertNextImg() {
   let insertedImg = document.createElement('img');
   insertedImg.classList.add('image');
-  insertedImg.src = 'img/s0.jpg';
+  if (imageWrapper.firstElementChild.tagName === 'P') {
+    insertedImg.src = 'img/s0.jpg';
+  } else {
+    let currentImgSrc = imageWrapper.firstElementChild.src
```

Comment:
those 3 variables will be visible only in block from `else`. So you don't have a chance to change them. So, they should be `const`